### PR TITLE
update magma rules to match tacops

### PR DESCRIPTION
**Pull request by NickAragua**
Makes the following changes to current megamek behavior regarding magma (TacOps page 36):

- hovercraft do not collapse magma crust, nor is the user nagged about it any longer
- jumping onto magma crust breaks through on 4+, added appropriate nag
- added nag for jumping into liquid magma
- consolidated "magma crust breaking" logic

Addresses #1093, although it's a little bit better for hovertanks.

Only thing I'm not sure about is how jumping + hover movement interplays here - if, let's say, a Kanga were to jump onto a lava crust, does it have a chance to break through?

### CODE REVIEW

File: megamek/i18n/megamek/common/report-messages.properties [hunk 1]
```diff
@@ -195,6 +195,7 @@
 2388=Have a nice trip!
 2390=<data> is landing on top of <data>!
 2395=<newline><data> (<data>) breaks through magma crust on a 6, rolls <data>.
```
Comment (sixlettervariables) [inline megamek/i18n/megamek/common/report-messages.properties:197]: Noticed this line doesn't match it's nag. Is it a 6+ or a 6?

**Review by sixlettervariables — APPROVED** [review]
One comment about related-ish messages.

File: megamek/i18n/megamek/common/report-messages.properties (hunk 1 above)
Comment (NickAragua) [reply megamek/i18n/megamek/common/report-messages.properties:197]: It's a 1d6, so there's no +

Comment (sixlettervariables) [reply megamek/i18n/megamek/common/report-messages.properties:197]: The nag states 6+

Comment (NickAragua) [reply megamek/i18n/megamek/common/report-messages.properties]: Ah, I see. Updated.

**Pull request merged**